CHANGELOG.md

```diff
@@ -1,18 +1,83 @@
 # Changelog
 
-## Unreleased (2017-10-17)
+## v0.8.0 (2017-10-22)
+
+We've got some big changes in v0.8.0 of Glimmer.js! The most notable change is
+that we have migrated from dasherized components (`x-profile`) to capitalized
+components (`Profile`). Multi-word components should be capitalized too, so
+`user-profile` becomes `UserProfile`.
+
+To upgrade existing Glimmer.js applications to 0.8.0, perform the following steps:
+
+1. Modify your application's `package.json`:
+  1. `@glimmer/application` should be `^0.8.0`.
+  2. `@glimmer/component` should be `^0.8.0` (you may need to add this
+     dependency).
+2. Rename component files from dasherized to `CapitalCase`. For example, the
+     `src/ui/components/user-profile` directory should be renamed to
+     `src/ui/components/UserProfile`.
+3. Change all component invocations from dasherized to `CapitalCase`. For
+   example, change `<user-profile @user={{user}} />` to
+   `<UserProfile @user={{user}} />`.
+
+These are all of the changes that should be necessary to migrate an existing
+app. For more information on this and other changes in Glimmer.js, make sure to
+read the [Glimmer.js Progress
+Report](https://emberjs.com/blog/2017/10/10/glimmer-progress-report.html) blog
+post.
+
+Note that 0.8.0 lays the foundation for compiling to binary bytecode, but does
+not yet enable it. Expect this functionality to be enabled in a future release,
+now that the requisite version of the underlying Glimmer compiler and VM have
+been upgraded.
 
 #### :bug: Bug Fix
-* [#26](https://github.com/glimmerjs/glimmer.js/pull/26) Fix incompatibility with {{each}} and frozen objects. ([@tomdale](https://github.com/tomdale))
+* `@glimmer/application-test-helpers`, `@glimmer/application`, `@glimmer/component`
+  * [#26](https://github.com/glimmerjs/glimmer.js/pull/26) Fix incompatibility with {{each}} and frozen objects. ([@tomdale](https://github.com/tomdale))
 
-#### Committers: 1
+#### :house: Internal
+* Other
+  * [#32](https://github.com/glimmerjs/glimmer.js/pull/32) Update "lerna-changelog" to v0.7.0. ([@Turbo87](https://github.com/Turbo87))
+  * [#27](https://github.com/glimmerjs/glimmer.js/pull/27) Add Changelog. ([@Turbo87](https://github.com/Turbo87))
+* `@glimmer/application`, `@glimmer/component`
+  * [#30](https://github.com/glimmerjs/glimmer.js/pull/30) Support running tests in the browser, Node, or both. ([@tomdale](https://github.com/tomdale))
+* `@glimmer/application`, `@glimmer/component`, `@glimmer/local-debug-flags`
+  * [#29](https://github.com/glimmerjs/glimmer.js/pull/29) Run tests in production and development modes. ([@tomdale](https://github.com/tomdale))
+
+#### Committers: 2
+- Tobias Bieniek ([Turbo87](https://github.com/Turbo87))
+- Tom Dale ([tomdale](https://github.com/tomdale))
+
+
+## v0.8.0-alpha.1 (2017-10-17)
```

Comment:
unfortunately https://github.com/lerna/lerna-changelog/issues/58 is still open, which means the order might be a little broken as visible here (`alpha.1`, `alpha.10`, `alpha.9`, ...)